setup.py

```diff
@@ -31,7 +31,7 @@
 
 requirements = [
     "numba==0.55.0",
-    "numpy>=1.21.0",
+    "numpy>=1.18.0,<=1.22.0",
```

Comment:
I think this actually needs to be 
`"numpy>=1.18.0,<1.22.0"`
because I get the error with this PR

```
Installed /home/rave/aquacrop-streamlit/.venv/lib/python3.9/site-packages/numpy-1.22.0-py3.9-linux-x86_64.egg
error: numpy 1.22.0 is installed but numpy<1.22,>=1.18 is required by {'numba'}
```

Reply:
Thanks, yes. Pushed the update